internal/sqladapter/testing/adapter.go.tpl

```diff
@@ -1092,15 +1092,14 @@ func TestBatchInsert(t *testing.T) {
 
 		go func() {
 			for i := 0; i < totalItems; i++ {
-				batch.Values(fmt.Sprintf("artist-%d", i))
+				batch.Values <- fmt.Sprintf("artist-%d", i)
```

Comment:
Yes I think `Values()` and `Done()` could work better here, as we don't really need to user to send anything to this channel directly.